wiki/Reporting_Bad_Behaviour/Abuse/en.md

```diff
@@ -0,0 +1,76 @@
+
+To do so, click the username of any GMT member in the list on that page. This will take you to their profile. If you are unsure of who to click, just use [this link instead](https://osu.ppy.sh/users/102335).
+
+Click the button with an envelope in it just below the user's country flag. This will take you to the chat screen.
+
+Paste a link to the forum post that contains the problematic content and tell the GMT a little bit about what's going on. They will remove the post for you as soon as they are able.
+
+You can also report the post by opening up your game and typing `!report <person> <problem>`, where  `<person>` is the username of the person causing you problems, and `<problem>` is the link to the post followed by a short explanation. If there are spaces in the person's username, please replace them with underscores (`_`).
+
+If the post still isn't gone within a reasonable timeframe, please send an email to [abuse@ppy.sh](mailto:abuse@ppy.sh) containing as much information as possible.
+
+### On a public osu!–related Discord server
+
+Immediately block the user by right-clicking their username and selecting `Block` in the drop-down list that appears.
+
+Afterwards, contact the administrators of the server at once by looking for someone at the very top of the users list and let them know what's happened. If you can't figure out who is an administrator, look for someone with a crown next to their name - they are the server owner and are responsible for what happens within it.
+
+Should this abuse continue and the administrators do nothing about it, [please notify the Discord Trust & Safety team by filing out a report](https://dis.gd/request).
+
+In addition, if you know the osu! username of the person bothering you, please send an email to [abuse@ppy.sh](mailto:abuse@ppy.sh) containing as much information as possible and it will be investigated promptly.
+
+### On a Twitter linked to an osu! user's profile
+
+Click the three dots in the top right of the problem tweet and click 'Report Tweet'. Make sure to follow the prompts and read them carefully. You might also want to [read this link](https://help.twitter.com/en/safety-and-security/report-abusive-behavior) for more information on the process.
+
+Afterwards, either block or mute the user in question by clicking the three dots in the top right of their tweet (if you're using the website or mobile client). Consult [this link](https://help.twitter.com/en/using-twitter/blocking-and-unblocking-accounts) for more information.
+
+Please be aware that the person you block will be aware that you have blocked them if they check your Twitter profile. If you are worried this will make the problem worse elsewhere, mute them instead. This will still remove their tweets from your timeline and prevent you from seeing any more content from them, but they will still be able to see all the tweets you make and can comment on them.
```

Comment:
a simplified version that reads better
```suggestion
Please be aware that the person will see you have blocked them if they check your Twitter profile. If you are worried this will make the problem worse elsewhere, mute them instead. This will remove their tweets from your timeline and prevent you from seeing any more content from them, but they will still be able to see and comment on all the tweets you make.
```